app/src/main/java/org/simple/clinic/contactpatient/ContactPatientUiRenderer.kt

```diff
@@ -35,12 +42,30 @@ class ContactPatientUiRenderer(
   private fun renderCallResultOutcome(model: ContactPatientModel) {
     if (model.hasCallResult) {
       val callResult = model.callResult!!.get()
-      ui.showCallResult(callResult.outcome, callResult.timestamps.updatedAt)
+      ui.showCallResult()
+      setupCallResultOutcomeUI(model, callResult)
+      ui.setCallResultUpdatedAtDate(callResult.timestamps.updatedAt.toLocalDateAtZone(clock.zone))
     } else {
       ui.hideCallResult()
     }
   }
 
+  private fun setupCallResultOutcomeUI(model: ContactPatientModel, callResult: CallResult) {
+    when (callResult.outcome) {
+      AgreedToVisit -> ui.setupAgreedToVisitCallResultOutcome()
+      RemindToCallLater -> setupRemindToCallLaterCallResultOutcome(model.appointment.remindOn)
+      RemovedFromOverdueList -> ui.setupRemovedFromListCallResultOutcome(callResult.removeReason!!)
+      is Unknown -> {}
+    }
+  }
+
+  private fun setupRemindToCallLaterCallResultOutcome(remindOn: LocalDate?) {
+    if (remindOn != null) {
+      val duration = Days.between(LocalDate.now(clock), remindOn)
+      ui.setupRemindToCallLaterCallResultOutcome(duration.amount)
```

Comment:
This API is from threeten extra library and we haven't used it anywhere else before. For similar behavior, we can use APIs from `java.time`.

```kt
val days = LocalDate.now(clock).until(remindOn, ChronoUnit.DAYS)
```